Project.toml

```diff
@@ -1,7 +1,10 @@
 name = "DecisionTreeAndRandomForest"
 uuid = "2e1a1f2f-ab8e-43f0-b75f-c7407e5d21e0"
-authors = ["Marleen Lukei marleen.lukei@campus.tu-berlin.de>"]
+authors = ["Marleen Lukei <marleen.lukei@campus.tu-berlin.de>, Cedric Lenßen <c.lenssen@campus.tu-berlin.de>"]
```

Comment:
All our names should be here ?

Reply:
You are right, it was a default setting :)

Reply:
I didn't know your full names and tu berlin email addresses, otherwise i would have added them ofc :)